src/Sarif.UnitTests/Processors/Log/SarifLogExtensionTests.cs

```diff
@@ -70,30 +71,30 @@ public void RebaseUri_WorksAsExpected()
         public void AbsoluteUri_ReversesRebasedURIs()
```

Comment:
>AbsoluteUri_ReversesRebasedURIs [](start = 20, length = 31)

made this thing slightly more debuggable. we now iterate over each log and block on discovering problem.

this test is just brain dead, though. it produces two kinds of uris only, for files table and results. the randomness is simply around the count of logs, runs, files, results. Dumb. let's just do a better job of writing this test later so that it operates against comprehensive data. this thought is why i didn't push a test pattern improvement elsewhere, just updated the failing test (which now passes).